packages/elements/src/components/ino-card/ino-card.tsx

```diff
@@ -15,6 +15,11 @@ export class Card implements ComponentInterface {
    */
   @Prop() inoSelected: boolean = false;
 
+  /**
+   * Disables the hover effect if true
+   */
+  @Prop() inoDisableHover: boolean = false;
```

Comment:
Maybe we should call it `inoDisableElevation`? That's how it's called in [Material-UI](https://material-ui.com/components/buttons/#contained-buttons)

Reply:
That's better. I'll rename it.